src/main/java/TongFibonacci.java

```diff
@@ -0,0 +1,17 @@
+public class TongFibonacci {
+
+    public int tongfibo(int n) {
+        int f1 = 0;
+        int f2 = 1;
+        int s = 0;
+        int count = 1;
+        while (count <= n) {
```

Comment:
Nên viết thành vòng for và đặt tên s thành sum

Reply:
done